components/javascript/toast.js

```diff
@@ -0,0 +1,31 @@
+var topToast = document.querySelector("#top-toast-id");
+var topToastContent = document.querySelector('#top-content');
+var topToastClose = document.querySelector('#top-toast-close');
+
+var bottomToast = document.querySelector('#bottom-toast-id');
+var bottomToastContent = document.querySelector('#bottom-content');
+var bottomToastClose = document.querySelector('#bottom-toast-close');
+
+function openTopToast(){
+    topToastContent.style.display='block';
```

Comment:
Use `setTimeout` to make the toast close itself after a few seconds since that is the default behaviour of a toast.

Reply:
Added setTimeout to close toast. Thank you for suggestion